### PR TITLE
Update nmt.py

### PR DESCRIPTION
**Pull request by MohammadJavadD**
It has issues when a path is provided. With these changes, it can handle a given path and look for the dataset in the path or download it.

### CODE REVIEW

**Comment by bruAristimunha** [comment]
Hey @MohammadJavadD!

Can fill the fix the test, please?

**Comment by bruAristimunha** [comment]
When everything turns green we can apply the merge

**Comment by MohammadJavadD** [comment]
@bruAristimunha done :) Everything is green now, except what's new which might not be necessary for these small changes.

File: braindecode/datasets/nmt.py [hunk 1]
```diff
@@ -84,16 +84,22 @@ class NMT(BaseConcatDataset):
...
     ):
-        # If the path is not informed, we fetch the dataset from zenodo.
-        if path is None:
+        # correct the path if needed
+        if path is not None:
+            look_4Labels_csv = glob.glob(f"{path}/**/Labels.csv", recursive=True)
+            if isinstance(look_4Labels_csv, list) and len(look_4Labels_csv) > 0:
+                path = Path(look_4Labels_csv[0]).parent
+
+        if path is None or len(look_4Labels_csv) == 0:
```
Comment (bruAristimunha) [inline braindecode/datasets/nmt.py:99]:
```suggestion
            list_csv = glob.glob(f"{path}/**/Labels.csv", recursive=True)
            if isinstance(list_csv, list) and len(look_4Labels_csv) > 0:
                path = Path(list_csv[0]).parent

        if path is None or len(list_csv) == 0:
```

**Comment by bruAristimunha** [comment]
Nice @MohammadJavadD,

I made one small modified in one variable to follow the PEP convention.

File: braindecode/datasets/nmt.py [hunk 2]
```diff
@@ -84,16 +84,22 @@ class NMT(BaseConcatDataset):
...
+        recording_ids=None,
+        preload=False,
+        n_jobs=1,
     ):
-        # If the path is not informed, we fetch the dataset from zenodo.
-        if path is None:
+        # correct the path if needed
+        if path is not None:
+            list_csv = glob.glob(f"{path}/**/Labels.csv", recursive=True)
+            if isinstance(list_csv, list) and len(look_4Labels_csv) > 0:
```
Comment (bruAristimunha) [inline braindecode/datasets/nmt.py:96]:
```suggestion
            if isinstance(list_csv, list) and len(list_csv) > 0:
```

**Comment by bruAristimunha** [comment]
Thank you @MohammadJavadD

**Pull request merged**